DataMigration/src/main/java/org/sagebionetworks/dian/datamigration/BridgeJavaSdkUtil.java

```diff
@@ -295,4 +337,124 @@ protected static ReportData makeReportData(String clientData) {
+
+        Map<String, String> migratedAttributes = new HashMap<>();
+        for (String attrKey : participant.getAttributes().keySet()) {
+            // Skip verification code and isMigrated attributes
+            if (!attrKey.equals(ATTRIBUTE_VERIFICATION_CODE) &&
+                    !attrKey.equals(ATTRIBUTE_IS_MIGRATED)) {
+                migratedAttributes.put(attrKey, participant.getAttributes().get(attrKey));
+            }
+        }
+
+        String arcId = migratedAttributes.get(ATTRIBUTE_ARC_ID);
+        if (arcId.length() != 6) {
+            throw new IllegalStateException("ARC_ID attribute is not 6 characters long");
+        }
+        String password = PasswordGenerator.INSTANCE.nextPassword();
+        migratedAttributes.put(ATTRIBUTE_VERIFICATION_CODE, password);
+
+        String studyId = participant.getStudyIds().get(0);
+
+        SignUp signUp = new SignUp()
+                .externalIds(ImmutableMap.of(studyId, arcId))
+                .password(password)
+                .dataGroups(new ArrayList<>())
+                .sharingScope(SharingScope.ALL_QUALIFIED_RESEARCHERS)
+                .attributes(migratedAttributes);
+
+        System.out.println("Creating participant account on bridge " + arcId);
+        String userId = researcherApi.createParticipant(signUp).execute().body().getIdentifier();
+
+        System.out.println("Writing availability report");
+        reportsApi.addParticipantReportRecordV4(userId, AVAILABILITY_REPORT_ID,
+                makeReportData(availability)).execute();
+
+        System.out.println("Writing test schedule report");
+        reportsApi.addParticipantReportRecordV4(userId, TEST_SCHEDULE_REPORT_ID,
+                makeReportData(testSchedule)).execute();
+
+        System.out.println("Writing completed tests report");
+        reportsApi.addParticipantReportRecordV4(userId, COMPLETED_TESTS_REPORT_ID,
+                makeReportData(completedTests)).execute();
+
+        System.out.println("Setting Device ID account IS_MIGRATED set to true...");
+        Map<String, String> deviceIdAttributes = new HashMap<>();
+        for (String key : participant.getAttributes().keySet()) {
+            if (key.equals(ATTRIBUTE_IS_MIGRATED)) {
+                deviceIdAttributes.put(ATTRIBUTE_IS_MIGRATED, ATTRIBUTE_VALUE_TRUE);
+            } else {
+                deviceIdAttributes.put(key, participant.getAttributes().get(key));
+            }
+        }
+        StudyParticipant updatedDeviceIdParticipant = new StudyParticipant();
+        updatedDeviceIdParticipant.setAttributes(deviceIdAttributes);
+        researcherApi.updateParticipant(participant.getId(), updatedDeviceIdParticipant).execute();
+
+        System.out.println("User successfully migrated");
+    }
+
+    /**
+     * @return  All users in all studies
+     * @throws IOException if something goes wrong
+     */
+    public static Map<String, List<String>> getAllUsers() throws IOException {
```

Comment:
`getAllUsers()` doesn't seem to be called anywhere.  Am I missing something?  If not, please remove this unused code.

Reply:
See https://github.com/Sage-Bionetworks/DianUserMigration-Java/pull/12 for this being used